src/test/cli/rbd/help.t

```diff
@@ -5,138 +5,155 @@
+                                           errors.
+      journal reset                        Reset image journal.
+      journal status                       Show status of image journal.
+      list (ls)                            List rbd images.
+      lock add                             Take a lock on an image.
+      lock list (lock ls)                  Show locks held on an image.
+      lock remove (lock rm)                Release a lock on an image.
+      merge-diff                           Merge two diff exports together.
+      migration abort                      Cancel interrupted image migration.
+      migration commit                     Commit image migration.
+      migration execute                    Execute image migration.
+      migration prepare                    Prepare image migration.
+      mirror image demote                  Demote an image to non-primary for RBD
+                                           mirroring.
+      mirror image disable                 Disable RBD mirroring for an image.
+      mirror image enable                  Enable RBD mirroring for an image.
+      mirror image promote                 Promote an image to primary for RBD
+                                           mirroring.
+      mirror image resync                  Force resync to primary image for RBD
+                                           mirroring.
+      mirror image snapshot                Create RBD mirroring image snapshot.
+      mirror image status                  Show RBD mirroring status for an image.
+      mirror pool demote                   Demote all primary images in the pool.
+      mirror pool disable                  Disable RBD mirroring by default
+                                           within a pool.
+      mirror pool enable                   Enable RBD mirroring by default within
+                                           a pool.
+      mirror pool info                     Show information about the pool
+                                           mirroring configuration.
+      mirror pool peer add                 Add a mirroring peer to a pool.
+      mirror pool peer bootstrap create    Create a peer bootstrap token to
+                                           import in a remote cluster
+      mirror pool peer bootstrap import    Import a peer bootstrap token created
+                                           from a remote cluster
+      mirror pool peer remove              Remove a mirroring peer from a pool.
+      mirror pool peer set                 Update mirroring peer settings.
+      mirror pool promote                  Promote all non-primary images in the
+                                           pool.
+      mirror pool status                   Show status for all mirrored images in
+                                           the pool.
+      mirror snapshot schedule add         Add mirror snapshot schedule.
+      mirror snapshot schedule list (... ls)
```

Comment:
Nit: might be nice to adjust the wrap point down so that we don't start putting most of the config actions on two lines above.